Formula/a/aicommit.rb

```diff
@@ -0,0 +1,20 @@
+class Aicommit < Formula
+  desc "AI-powered commit message generator"
+  homepage "https://github.com/coder/aicommit"
+  url "https://github.com/coder/aicommit/archive/refs/tags/v0.6.3.tar.gz"
+  sha256 "f42fac51fbe334f4d4057622b152eff168f4aa28d6da484af1cea966abd836a1"
+  license "CC0-1.0"
+
+  depends_on "make" => :build
+  depends_on "go" => "1.21"
+
+  def install
+    ENV["VERSION"] = version
+    system "make", "build"
+    bin.install "bin/aicommit"
+  end
+
+  test do
+    assert_match "aicommit v#{version}", shell_output("#{bin}/aicommit version")
```

Comment:
We can keep the version check (since we want to check that the `ldflags` that we set in `install` are correct), but:

Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I'm not sure happy with this test either but the nature of the tool is that any substantial functionality requires an API key to an external service. Since the Go binary is self-contained there is little risk that of the `aicommit version` working but the rest of the functionality having a substantial failure.

Reply:
You could try connecting to an external service without an API key (or with an invalid one) and check that it errors out as expected.

Reply:
[6f10403](https://github.com/Homebrew/homebrew-core/pull/192146/commits/6f10403b0bf45e3cbb4f090579e8bade9060d16a) attempts this.. this is my first experience with homebrew and ruby so please excuse any funkiness.